client/asset/dcr/rpcwallet.go

```diff
@@ -912,26 +912,36 @@ func (w *rpcWallet) Tickets(ctx context.Context) ([]*asset.Ticket, error) {
 			w.log.Errorf("GetTransaction error for ticket %s: %v", h, err)
 			continue
 		}
+		blkHash, err := chainhash.NewHashFromStr(tx.BlockHash)
```

Comment:
How would this work for unmined tickets? Or are all tickets guaranteed to be included in a block?

Reply:
I believe it errors and I saw this while testing with a block hash of all zeros....

Reply:
Testing out on simnet, it seems vsp is only including mined tickets in the results. @buck54321 do we need confirming tickets for native wallets? Or we can ignore these here as well so they act the same. Probably we want the unconfirmed tickets in results...

Reply:
I saw that too, but I figured we could follow up since it's not particular to RPC staking.

Reply:
Will set unmined block heights to -1 here and leave a TODO in the vsp wallet logic.